packages/candy_editor/src/assets/formTypes/BoolForm/converters.ts

```diff
@@ -0,0 +1,13 @@
+import { CandyBool } from '../../../types';
+
+export function convertToBool(selectedValue: string): CandyBool | undefined {
```

Comment:
Please change to convertToCandyBool and update all other converter functions to include Candy.